File: Recursion/printname.java
import java.util.*;

public class printname {
    public static void main(String args[]) {
        Scanner sr = new Scanner(System.in);
        int n = sr.nextInt();
        System.out.println(n);
        name(1, n);

        sr.close();
    }

    static void name(int i, int n) {
        if (i > n)
            return;
        System.out.println("Bhakti");
        name(i + 1, n);
    }

}

File: RecursionInCpp/printName.cpp
#include<iostream>
using namespace std;
int printName(int name){
    if(name < 1){
        return -1;
    }else{
        cout<<"Bhakti ";
        printName(name-1);
        return 0;
    }
}
int main(){
    int n;
    cin>>n;
    printName(n);
    return 0;
}